sympy/stats/joint_rv_types.py

```diff
@@ -177,11 +180,25 @@ def MultivariateT(syms, mu, sigma, v):
     Parameters
     ==========
 
-    syms: list/tuple/set of symbols for identifying each component
-    mu: A list/tuple/set consisting of k means,represents a k
+    sym: A symbol/str for identifying the random variable. This can be
+        indexed to obtain each component of the joint probability space.
+    mu: A list/matrix consisting of k means,represents a k
```

Comment:
I think the comma here is supposed to be a slash, but it would be clearer to just delete either "means" or "represents"

Reply:
I would delete "means". "Represents" is a better word here.